Cargo.toml

```
@@ -403,6 +403,17 @@ description = "Renders a 2d mesh"
 category = "2D Rendering"
 wasm = true
 
+[[example]]
+name = "mesh2d_arcs"
+path = "examples/2d/mesh2d_arcs.rs"
+doc-scrape-examples = true
+
+[package.metadata.example.mesh2d_circular]
```

Forgot to update this too
```suggestion
[package.metadata.example.mesh2d_arcs]
```